### ISSUE TITLE
Plex MCM Agent Ignores <collection> tag

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1631]
Originally reported on Google Code with ID 1636

```
What steps will reproduce the issue?
1. Edit collections/set field in MCM metadata tab for a title.
2. Save metadata changes, verify movie.xml contains <collection> tag with content.
3. Force refresh of title within plex with MCM agent selected for that library.
4. Title within plex still shows empty collections field.


What is the expected output?  What do you see instead?
Expected output is collections field within plex should reflect what was entered in
MCM.
Instead an empty field is displayed in plex.

What is the exact version of the Media Center Master that you are using?
2.13 build 35014.727
Agent version 1.01


Relevant screenshots attached.
Also forum thread for reference.
http://forums.mediacentermaster.com/viewtopic.php?f=4&t=10519&p=41464#p41464

```

Reported by `mrjamesroe` on 2015-01-30 19:43:36

<hr>
- _Attachment: xml.png<br>![xml.png](https://storage.googleapis.com/google-code-attachments/mediacentermaster/issue-1636/comment-0/xml.png)_
- _Attachment: plex.png<br>![plex.png](https://storage.googleapis.com/google-code-attachments/mediacentermaster/issue-1636/comment-0/plex.png)_

**Comment by psouza4** [comment]
Support for collections in the Plex scanner added in an upcoming release of Media Center Master.

**Comment by psouza4** [comment]
Released in version 2.18 build 4518.858 (2.18.4518.858).
